cpp/src/arrow/flight/client.h

```diff
@@ -73,6 +91,16 @@ class ARROW_FLIGHT_EXPORT FlightClientOptions {
   std::string private_key;
   /// \brief A list of client middleware to apply.
   std::vector<std::shared_ptr<ClientMiddlewareFactory>> middleware;
+  /// \brief A soft limit on the number of bytes to write when sending
+  ///     Arrow data to a server.
+  ///
+  /// Used to help limit server memory consumption. Only enabled if
+  /// positive. When enabled, \a FlightStreamWriter.Write* may yield a
```

Comment:
Why the `\a`s?

Reply:
I was under the mistaken impression that Doxygen uses them for monospace text, but it's just intended to annotate arguments, so I've removed them. https://www.doxygen.nl/manual/commands.html#cmda